src/Features/Core/Portable/Completion/CompletionItemRules.cs

```diff
@@ -51,5 +51,14 @@ internal partial class CompletionItemRules
         {
             return null;
         }
+
+        /// <summary>
+        /// Returns true if <paramref name="item"/> is a better match than <paramref name="other"/>.
```

Comment:
The thing that calls this has many criteria, of which is is one. False means it's worse. Null means it's equivalent/the check hasn't been implemented by this provider. (Should prob. doc this).